packages/commerce-sdk-react/src/hooks/ShopperGiftCertificates/query.ts

```diff
@@ -37,14 +38,21 @@ export const useGiftCertificate = (
     // Parameters can be set in `apiOptions` or `client.clientConfig`;
     // we must merge them in order to generate the correct query key.
     const netOptions = omitNullableParameters(mergeOptions(client, apiOptions))
+    // get param keys for the api from netOptions
+    const paramKeys = [
+        ...paramKeysMap[methodName],
+        ...getCustomKeys(netOptions.parameters)
+    ] as const
```

Comment:
[NIT] I noticed that some `query.ts` files define `paramKeys` with `as const` while others don't. We should stick with one convention for consistency

Reply:
good catch. I've removed `as const` for the paramKeys